ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplStatsTracker.java

```diff
@@ -36,7 +36,7 @@
 public class ReplStatsTracker {
 
   // Maintains the length of the RM_Progress column in the RDBMS, which stores the ReplStats
-  public static int RM_PROGRESS_LENGTH = 24000;
+  public static int RM_PROGRESS_LENGTH = 10000;
```

Comment:
Were you able to justify the size?

Reply:
Can you please add a sample metric with top K=5/K=10 both

Reply:
Attached the sample outputs and size enhancements in comment section.